wizelineacademy/laboratoria-hacker-books
# Feat/fe-6/layout component

## What does this PR do?

Base structure and config layout component applied and separation some fundamental new components basic for test layout container.

## Where should the reviewer start?

- components/footer/footer.module.css - Some base styles footer 
- components/footer/footer.stories.tsx - Created storybook footer component
- components/footer/footer.tsx - Separated footer component in a component for test in config layout component
- components/layout/layout.stories.tsx - Created storybook layout component
- components/layout/layout.tsx - Created and setup base layout component where called Navbar, Sidebar, Main and Footer
- components/main/main.tsx - Base setup for children in component main container
- components/sidebar/sidebar.tsx - Created new test sidebar component for test in config layout component
- pages/_app.tsx - Called to the app layout component


## How should this be manually tested?

- Pull change branch feat/laboratoria-FE-6/layout-component
- Check new home components were separate footer
-  Verify new component only for a test layout component Sidebar, Main here is (children) for all content to the main container
- Review layout component base config for Layout
- On page review that is Navbar, Sidebar, Main (child home) and footer

:construction:

## What are the relevant tickets?

https://trello.com/c/zHQGsekh/9-fe-6create-layout-component

## Screenshot 

- Web

![Screen Shot 2020-12-07 at 13 23 56](https://user-images.githubusercontent.com/56928486/101395393-7dfacf80-388f-11eb-8fb7-37ec00aae8f8.png)


- Storybook the styles does not take the styles because they are not tailwind but this in main container will change

![Screen Shot 2020-12-07 at 12 43 43](https://user-images.githubusercontent.com/56928486/101395262-50158b00-388f-11eb-8032-38b619185cdf.png)


## Questions
Could you review this PR @ivanovishado @jonathanperazawl, please?

🔢
